app/views/layouts/_top_bar.html.erb

```diff
@@ -47,7 +47,7 @@
         </a>
 
         <div class="crayons-header__menu" id="crayons-header__menu">
-          <button type="button" class="crayons-header__menu__trigger navigation-butt" id="navigation-butt" aria-label="Navigation menu">
+          <button type="button" class="crayons-header__menu__trigger" id="member-menu-button" aria-label="Navigation menu">
```

Comment:
It makes sense to rename this, but I will add that `navigation-butt` made me chuckle every time I saw it.